docs/api/enterprise.md

```diff
@@ -1600,7 +1600,8 @@ curl -X PATCH http://coder-server:8080/api/v2/scim/v2/Users/{id} \
   "roles": [
     {
       "display_name": "string",
-      "name": "string"
+      "name": "string",
+      "organization_id": "string"
```

Comment:
We should probably also drop a comment here regarding the experimental status of this thing until it's ready for prime time

Reply:
I wish the docs would read the `omitempty` field like our type generator does. No organization roles are currently returned by any api in use by our frontend, so the field is always excluded in the json responses.